sim/paladin/retribution/retribution.go

```diff
@@ -72,6 +73,11 @@ func (ret *RetributionPaladin) RegisterSpecializationEffects() {
 	mhWeapon := ret.GetMHWeapon()
 	if mhWeapon != nil && mhWeapon.HandType == proto.HandType_HandTypeTwoHand {
 		ret.PseudoStats.SchoolDamageDealtMultiplier[stats.SchoolIndexPhysical] *= 1.25
+		ret.AddStaticMod(core.SpellModConfig{
+			Kind:       core.SpellMod_DamageDone_Flat,
+			ClassMask:  paladin.SpellMaskJudgement,
+			FloatValue: 0.25,
+		})
```

Comment:
If there is a judgement that deals physical damage this and the above line would double dip this bonus. Not sure if that is possible but just worth mentioning

Reply:
Hmm, they all deal holy damage but using melee hit/crit